src/Microsoft.Health.Fhir.Shared.Core/Features/Validation/ProfileValidator.cs

```diff
@@ -38,12 +39,20 @@ public ProfileValidator(IProvideProfilesForValidation profilesResolver, IOptions
 
         private Validator GetValidator()
         {
+            var expanderSettings = new ValueSetExpanderSettings
+            {
+                MaxExpansionSize = 20000, // Set your desired max expansion size here
```

Comment:
Can we set this from a configuration somewhere?

Reply:
We could add a setting, and make it configurable.  But I wanted a quick change to address this issue.  Then we can consider a more in depth change.